packages/@o3r/eslint-config-otter/schematics/ng-add/linter/index.ts

```diff
@@ -120,9 +120,48 @@ export function updateLinterConfigs(options: { projectName?: string | null | und
     tree.overwrite('/angular.json', JSON.stringify(workspace, null, 2));
   };
 
+  /**
+   * Handle linter errors to comply with Otter rules
+   * @param tree
+   * @param context
+   */
+  const handleOtterEslintErrors: Rule = async (tree: Tree, context: SchematicContext) => {
+    if (!options.projectName) {
+      return;
+    }
+    const { getWorkspaceConfig } = await import('@o3r/schematics');
+    const workspace = getWorkspaceConfig(tree);
+    if (!workspace) {
+      return;
+    }
+    const workspaceProject = workspace.projects[options.projectName];
+
+    if (!workspaceProject) {
+      context.logger.warn('No project detected, linter task can not be added.');
+      return;
+    }
+    const projectRoot = workspaceProject.root;
+    const appComponentPath = posix.join(projectRoot, 'src/app/app.component.ts');
+    if (tree.exists(appComponentPath)) {
+      const appComponentContent = tree.readText(appComponentPath);
+      const newAppComponentContent = appComponentContent.replace(/(?<!public\s|private\s|protected\s)(^\s*)(\b\w+\b\s?[=:].*;)/gm, `$1public $2`);
```

Comment:
I feel like we are implementing an eslint rule fix here which seem to be the wrong place